go.mod

```diff
@@ -34,4 +34,6 @@ require (
 	sigs.k8s.io/controller-runtime v0.8.3
 )
 
+replace github.com/codeready-toolchain/toolchain-common => github.com/xcoulon/toolchain-common v0.0.0-20210921100839-b43561751e72
```

Comment:
Reminder

Reply:
removed in https://github.com/codeready-toolchain/registration-service/pull/216/commits/f7a0b785647295320fb1d46ae3ba35a29296f90d